src/main/java/bankservice/demo/repository/AccountRepository.java

```diff
@@ -0,0 +1,14 @@
+package bankservice.demo.repository;
+
+import bankservice.demo.entity.Account;
+import java.util.List;
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+public interface AccountRepository extends JpaRepository<Account, Long> {
+    @Query("FROM Account a WHERE a.user.phoneNumber = ?1")
+    List<Account> getByPhoneNumber(String phoneNumber);
```

Comment:
```suggestion
    List<Account> getByUserPhoneNumber(String phoneNumber);
```
will this work without `@Query` annotation?

Reply:
With this naming, it works without `@Query`

Reply:
Fixed